utils/osdetail/version_windows.go

```diff
@@ -29,6 +30,8 @@ func WindowsNTVersion() (string, error) {
 	if !fetched {
 		_, _, windowsNTVersion, err = host.PlatformInformation()
 
+		windowsNTVersion = strings.Split(windowsNTVersion, " Build ")[0]
```

Comment:
I suggest only splitting with `" "` as this will still work if something changes in the future. The version string itself should not have any spaces in it anyway.

Reply:
Also, for improved performance, check out `strings.SplitN`.

Reply:
Still, I will need SplitN(..., 2)[0] which shouldn't change Performance in my example. If I just check for " ", then for sure it matters.